frontend/website/src/components/closeSidebar.svelte

```diff
@@ -0,0 +1,6 @@
+<svg width="1em" height="1em" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
replace 1em with 0.7em

Reply:
why?

Reply:
The advantage of having it 1em means it scales with `font-size:` by an exact factor of the font size.

Reply:
yeah but it just looks straight up bad

Reply:
test in discord <@!383195095610163200>

Reply:
im confused how does it look bad? Changing both values to 0.7em means that it scales slightly less than a factor of `font-size` and changing the one value (height) to `0.7em` would make it not a perfect square. By making it a perfect square it just pads the region which is slightly smaller defined in the view box.

Reply:
if you think the size of the icon should be smaller you should have just said that and we can change it on the side nav and the chat room directly.

Reply:
lol, yeah, sorry, small brain moment